rotkehlchen/data_import/importers/bittrex.py

```diff
@@ -40,52 +52,160 @@ def _consume_trades(
-            timestamp=deserialize_timestamp_from_date(
-                date=csv_row['Time (UTC)'],
-                formatstr=timestamp_format,
-                location='Bittrex order history import',
-            ),
-            location=Location.BITTREX,
-            base_asset=asset_from_bittrex(csv_row['Base']),
-            quote_asset=quote_asset,
-            trade_type=TradeType.BUY if csv_row['Transaction'] == 'Bought' else TradeType.SELL,
-            amount=deserialize_asset_amount(csv_row['Quantity (Base)']),
-            rate=deserialize_price(csv_row['Price']),
-            fee=deserialize_fee(csv_row['Fees (Quote)']),
-            fee_currency=quote_asset,
-            link=f'Imported bittrex trade {csv_row["TXID"]} from csv',
-            notes=csv_row['Notes'],
-        )
+        if file_type == BittrexFileType.TRADES:
+            quote_asset = asset_from_bittrex(csv_row['Quote'])
+            return Trade(
+                timestamp=deserialize_timestamp_from_date(
+                    date=csv_row['Time (UTC)'],
+                    formatstr=timestamp_format,
+                    location='Bittrex order history import',
+                ),
+                location=Location.BITTREX,
+                base_asset=asset_from_bittrex(csv_row['Base']),
+                quote_asset=quote_asset,
+                trade_type=TradeType.BUY if csv_row['Transaction'] == 'Bought' else TradeType.SELL,
+                amount=deserialize_asset_amount(csv_row['Quantity (Base)']),
+                rate=deserialize_price(csv_row['Price']),
+                fee=deserialize_fee(csv_row['Fees (Quote)']),
+                fee_currency=quote_asset,
+                link=f'Imported bittrex trade {csv_row["TXID"]} from csv',
+                notes=csv_row['Notes'],
+            )
+        elif file_type == BittrexFileType.TRADES_OLD:
+            quote_asset_symbol, base_asset_symbol = csv_row['Exchange'].split('-')
+            return Trade(
+                timestamp=deserialize_timestamp_from_date(
+                    date=csv_row['Closed'],
+                    formatstr='%m/%d/%Y %I:%M:%S %p',
+                    location='Bittrex order history import',
+                ),
+                location=Location.BITTREX,
+                base_asset=asset_from_bittrex(base_asset_symbol),
+                quote_asset=asset_from_bittrex(quote_asset_symbol),
+                trade_type=TradeType.BUY if csv_row['Type'] == 'LIMIT_BUY' else TradeType.SELL,
+                amount=deserialize_asset_amount(csv_row['Quantity']),
+                rate=deserialize_price(csv_row['Price']),
+                link=f'Imported bittrex trade {csv_row["OrderUuid"]} from csv',
+            )
+        else:
+            quote_asset = asset_from_bittrex(csv_row['QUOTE'])
+            return Trade(
+                timestamp=deserialize_timestamp_from_date(
+                    date=csv_row['CLOSED'],
+                    formatstr='%m/%d/%Y %I:%M:%S %p',
```

Comment:
shouldn't you use here the timestamp format? It could be that it differs among users